test/parallel/test-http-client-default-headers-exist.js

```diff
@@ -47,8 +47,8 @@ const server = http.createServer(function(req, res) {
   const requestHeaders = Object.keys(req.headers);
   requestHeaders.forEach(function(header) {
     assert.notStrictEqual(
-      expectedHeaders[req.method].indexOf(header.toLowerCase()),
-      -1,
+      expectedHeaders[req.method].includes(header.toLowerCase()),
+      false,
```

Comment:
Ah missed that! Will fix it!
Also @gibfahn , why doesn't assert provide a functions like `assertFalse` ? 
(Similar to `assertok` )